manager/controllers/app/multicluster/multi_cluster_manager.go

```diff
@@ -0,0 +1,75 @@
+package multicluster
+
+import (
+	"fmt"
+	"github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+type ClusterManager interface {
+	GetClusters() ([]Cluster, error)
+	GetBlueprint(cluster string, namespace string, name string) (*v1alpha1.Blueprint, error)
+	CreateBlueprint(cluster string, blueprint *v1alpha1.Blueprint) (DeploymentDetails, error)
+	UpdateBlueprint(cluster string, blueprint *v1alpha1.Blueprint) (DeploymentDetails, error)
```

Comment:
Do we really need both Create and Update? (vs something that is similar to apply)

Reply:
This is an initial version. I think in future it makes sense to merge it once we know how razee really behaves.

Reply:
NP, you will definitely get to it when you implement the controller anyway